app/models/election.js

```diff
@@ -108,7 +112,7 @@ electionSchema.methods.addAlternative = async function (alternative) {
   return savedAlternative;
 };
 
-electionSchema.methods.addVote = async function (user, priorities) {
+electionSchema.methods.addVote = async function (user, priorities, accessCode) {
```

Comment:
This can be removed I guess?

Reply:
Correct, that was the initial implementation, where you passed the code along with the vote. It was a far simpler implementation, but it was annoying to go through the whole `vote and prioritize` then getting "Wrong Code" :) So I'll remove this.